__tests__/app.test.js

```diff
@@ -292,4 +292,29 @@ describe("/api/comments/:comment_id",()=>{
             expect(body.msg).toBe("Not found")
         })
     })
+})
+describe("/api/users",()=>{
+    test("GET-200 response with all users details",()=>{
+        return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({body})=>{
+            
```

Comment:
There is an assertion that needs to be here 👀 If the response body.users is an empty array, the assertions in the .forEach won't run and the test will automatically pass 🫨 So we need an assertion to check that the array has a `length` property.